src/control/lib/netdetect/netdetect.go

```diff
@@ -786,7 +789,7 @@ func mercuryToLibFabric(provider string) (string, error) {
 		return "gni", nil
 	default:
 	}
-	return "", errors.Errorf("fabric provider: %s not known by libfabric.  Use 'daos_server network list' to view supported providers", provider)
+	return "", errors.Errorf("fabric provider: %s not known by libfabric.  Use 'daos_server network scan -p all' to view supported providers", provider)
```

Comment:
I didn't notice these before, but while you're in there changing these messages... I understand that the intent was to be helpful to the user, but the netdetect library is the wrong place for this. Among other problems, it's incorrect when the caller is anything other than daos_server, and as you've discovered, it adds maintenance drag.

I suggest something more concise and generic, such as the following:
```go
	return "", errors.Errorf("unknown fabric provider %q", provider)
```

Reply:
Good idea.